### DISCUSSION TITLE
[2758000]   名利游戏 [仅本体]

### DISCUSSION THREAD

**Discussion by segadcss** [pjy612/SteamManifestCache#514] [Share]
`appid:2758000`
[2758001_2984932048339894641.zip](https://github.com/pjy612/SteamManifestCache/files/14375861/2758001_2984932048339894641.zip)

keys:
```
2758001;70d82cd1e81fe339eb087ae9438c39ee6e0540ab7f6efeaab136e4ca57f85199
```

**Comment by CSSZYF** [comment]
#26 格式不对

**Reply by segadcss** [reply]
已经在内测了，这个只分享

**Reply by CSSZYF** [reply]
希望不公开到仓库只在这里分享吗?

**Reply by pjy612** [reply]
那也麻烦 格式一致
